src/perftest_resources.c

```diff
@@ -1659,7 +1715,7 @@ int create_single_mr(struct pingpong_context *ctx, struct perftest_parameters *u
 	/* ODP */
 	#ifdef HAVE_EX_ODP
 	if (user_param->use_odp) {
-		if ( !check_odp_support(ctx) )
+		if ( !check_odp_support(ctx, user_param->verb, user_param->connection_type) )
```

Comment:
Thanks , looks great.
can you please pass only the user_param and use the pointers in check_odp_support?

Reply:
Thanks for the review!
I've done it.